### PR TITLE
Convert containerd presubmit to knative-build agent

### PR DESCRIPTION
**Pull request by krzyzacy**
ref https://github.com/kubernetes/test-infra/issues/10680

/assign @fejta @Random-Liu

### CODE REVIEW

File: config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml [hunk 1]
```diff
@@ -1,115 +1,135 @@
...
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
     labels:
       preset-service-account: "true"
-    spec:
-      containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190125-3d9554697e-master
+      preset-k8s-ssh: "true"
```
Comment (fejta) [inline config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml]: Does this work? I am not sure we verified this on the build controller

Comment (krzyzacy) [reply config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml]: :-) If you didn't make it to work, then it probably doesn't work

File: config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml [hunk 2]
```diff
@@ -1,115 +1,135 @@
 presubmits:
   containerd/cri:
```
Comment (Random-Liu) [inline config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml]: Can we keep the build job? It is still better to have a separate job for build.

We can of course still build again for each node e2e test.

**Comment by krzyzacy** [comment]
will send another PR to address @fejta's comment above

**Comment by krzyzacy** [comment]
rebased

**Comment by Random-Liu** [comment]
Can we get this done soon? We don't have presubmit node e2e in containerd repository now.

**Comment by krzyzacy** [comment]
lgtm?

**Pull request merged**